akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -511,19 +511,6 @@ class HttpExt private[http] (private val config: Config)(implicit val system: Ex
 
   /**
    * INTERNAL API
-   *
-   * Starts a new connection pool to the given host and configuration and returns a [[akka.stream.scaladsl.Flow]] which dispatches
-   * the requests from all its materializations across this pool.
-   * While the started host connection pool internally shuts itself down automatically after the configured idle
-   * timeout it will spin itself up again if more requests arrive from an existing or a new client flow
-   * materialization. The returned flow therefore remains usable for the full lifetime of the application.
-   *
-   * Since the underlying transport usually comprises more than a single connection the produced flow might generate
-   * responses in an order that doesn't directly match the consumed requests.
-   * For example, if two requests A and B enter the flow in that order the response for B might be produced before the
-   * response for A.
-   * In order to allow for easy response-to-request association the flow takes in a custom, opaque context
-   * object of type `T` from the application which is emitted together with the corresponding response.
```

Comment:
Why did you drop this?

Reply:
Because it's an old copy of the scaladoc seen above on the public `newHostConnectionPool`. No need to reproduce it for an internal method.